src/transporters.ts

```diff
@@ -102,11 +102,11 @@ export class DefaultTransporter implements Transporter {
         err.message = body.error.errors
           .map((err2: Error) => err2.message)
           .join('\n');
-        err.code = body.error.code;
+        err.code = body.error.code.toString();
         err.errors = body.error.errors;
       } else {
         err.message = body.error.message;
-        err.code = body.error.code || res.status;
+        err.code = body.error.code.toString() || res.status.toString();
```

Comment:
I think keeping the code & status as numbers is more conventional, e.g.:
- https://github.com/googleapis/google-auth-library-nodejs/issues/1092
- https://developer.mozilla.org/en-US/docs/Web/API/Response/status

Reply:
Fair, but it seems like Gaxios expects a string: https://github.com/googleapis/gaxios/blob/18a48fc298c8d0e707b7f6e5cd892cb7c239f1db/src/common.ts#L21

SHould we close this as WAI? https://github.com/googleapis/google-auth-library-nodejs/issues/1092